Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -329,7 +330,7 @@ def set_preprocessor(self, preproc):
     def handleNewSignals(self):
         """Reimplemented from OWWidget.handleNewSignals."""
         self._update_class_selection()
-        self.apply()
+        self.commit()
```

Comment:
I believe that at least when the schema is first loaded, all signals need to be processed.

Also, I think that when the widget receives data, it should process and output it. "Apply" button should refer to application of changes in the user interfaces. (This could be different if the name is "Commit" and "Send", which imply sending the data, not just applying changes.)